Sources/FluentKit/Concurrency/Database+Concurrency.swift

```diff
@@ -3,15 +3,15 @@ import NIOCore
 
 @available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
 public extension Database {
-    func transaction<T>(_ closure: @escaping (Database) async throws -> T) async throws -> T {
+    func transaction<T>(_ closure: @Sendable @escaping (Database) async throws -> T) async throws -> T {
```

Comment:
what's `@Sendable`? o_O

Reply:
Oh Tanner....

🤣

Reply:
In all seriousness this shouldn't be API breaking but does it add any extra warnings?

Reply:
@0xTim It actually corrected two of the four in that file. AFAIK it doesn't add any that wouldn't have showed up anyway for callers.